.eslintrc.json

```
@@ -1,16 +1,21 @@
 {
     "env": {
         "browser": true,
-        "webextensions": true
+        "webextensions": true,
+        "jest/globals": true
     },
     "extends": [
+        "plugin:jest/recommended",
         "plugin:import/recommended",
         "plugin:import/react",
         "plugin:promise/recommended",
         "plugin:react/recommended",
         "standard",
         "standard-react"
     ],
+    "globals": {
+        "db": true
+    },
```

`db` should not be a global. It is exposed as a global for console debugging, but using it as a global anywhere in the code is a dangerous mistake (it will break in production).

---

I thought so too at first. But if you refer to [line](https://github.com/WebMemex/webmemex-extension/pull/104/files#diff-cd67c91cd27eef6025852e6802ed2512R5) , it takes advantage of the jest mocking feature and uses db as a global variable. Removing it from the ```globals``` causes it to throw an error stating db is note defined.

---

Ok, not sure how the mock system works; I would find it weird if it adds such globals implicitly. Anyway, you could add a line `/* global db */` to the file to suppress the error. Or it should be in a config only used for tests, because it is important to have this warning enabled in the rest of the code.